addons/hr_holidays/models/resource.py

```diff
@@ -162,7 +162,7 @@ class ResourceCalendar(models.Model):
 
     def _compute_associated_leaves_count(self):
         leaves_read_group = self.env['resource.calendar.leaves']._read_group(
-            [('resource_id', '=', False), ('calendar_id', 'in', [False, *self.ids])],
```

Comment:
what is reason behind this change ?

Reply:
When clicking on the public time off smart button, it shows the public time off entries along with their associated public time off, including those where the working schedule is set to "false.", so we needed to change that to only show the associated public time off.
This field was counting the public time off entries with a "false" value, so I removed the "false" to ensure that only relevant public time off entries are counted.